l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -721,8 +721,12 @@ def setDatiPagamento(self, invoice, body):
     def setAttachments(self, invoice, body):
         if invoice.fatturapa_doc_attachments:
             for doc_id in invoice.fatturapa_doc_attachments:
+                nome_attachment = doc_id.datas_fname if \
+                   len(doc_id.datas_fname) <= 60 else \
+                   ''.join([doc_id.name.rsplit('.', 1)[0][:55],
+                            doc_id.name.rsplit('.', 1)[1]])
```

Comment:
I would use `os.path.splitext` to get the file name without extension

Reply:
ok, I add os import